packages/apollo-gateway/src/__tests__/integration/nockMocks.ts

```diff
@@ -0,0 +1,114 @@
+import nock from 'nock';
+
+export const mockLocalhostSDLQuery = () =>
```

Comment:
I think we should provide a way to override the results of these requests so we can test error handling better